_includes/yellow/yellow-reseat-cm5.md

```diff
@@ -28,8 +28,31 @@
      - It needs to run parallel to the Home Assistant Yellow board.
      - You can also pull a bit on the sides to make sure it is not loose.
 
+{% endstepContent %}
+{% endstep %}
+
+{% step "Placing the heat pads" %}
+{% image "/static/img/yellow/cm5-place-heat-pads-02.png" "Top view of the board, showing how to install the heat pads" %}
+{% stepContent %}
+
+1. If your Yellow came with a plastic bag labelled **CM5**, containing screws and heat pads, place those heat pads on the CM5.
+2. Align the pads as shown in the image.
+3. Make sure that the pad with the cut off corner is placed so that there is room for the screw.
+
+{% endstepContent %}
+{% endstep %}
+
+{% step "Optional: Adding screws" %}
+{% image "/static/img/yellow/cm5-place-heat-pads-and-screws-01.png" "Top view of the board, showing how to install the screws" %}
+{% stepContent %}
+
 {% include "fragments/notice-dont-use-screws.md" %}
 
+1. If the packaging included a plastic bag labelled **CM5** with screws and heat pads, you have the option to screw the CM5 module in place.
+2. Only add screws if you feel the module did not click in place properly.
```

Comment:
The screws press down on the corners, that usually doesn't help all that much if the module doesn't snap in properly. In this case, the user should remove and reinsert it again. But I would not explicitly mention that.

```suggestion
2. Screws are optional and can help keep the Compute Module in place during transportation.
```

Maybe this should be part of the notice? :thinking: